users/tests.py

```diff
@@ -12,39 +13,136 @@ def setUp(self):
         super().setUp()
         # unset client credentials token so that the subsequent API calls goes as guest
         self.client.credentials()
-        self.user_mobile = "+919876543210"
 
     def test_guest_can_request_for_otp(self):
         response = self.client.post(
-            reverse("request-otp"), {"mobile": self.user_mobile}
+            reverse("request-otp"), {"mobile": self.user.mobile}
```

Comment:
I can't see `user.mobile` being set in the BaseTestCase
Default is None, is it being set to None here?